README.md

```diff
@@ -14,6 +14,7 @@ Please refer to Prerequisites for dependencies and installation instructions
 1. Run `poetry run python manage.py collectstatic --noinput` to collect static files
 1. Run `poetry run python manage.py migrate` this will create a local sqlite3 database and migrate any tables
 1. Run `poetry run python manage.py waffle_switch search-sort-radio-buttons off --create` to setup the waffle switch tables
+1. Run `poetry run python manage.py waffle_switch display-result-tags off --create` to setup the waffle switch tables
```

Comment:
The comment is the same as the example above

Reply:
fixed